msteams-platform/toolkit/TeamsFx-SDK.md

```diff
@@ -592,33 +739,34 @@ This section provides several code snippets for other scenarios that are related
 
   1. Set the connection configuration.
 
-  ```typescript
-  // Equivalent to:
-  // const sqlConnectConfig = new DefaultTediousConnectionConfiguration({
-  //    sqlServerEndpoint: process.env.SQL_ENDPOINT,
-  //    sqlUsername: process.env.SQL_USER_NAME,
-  //    sqlPassword: process.env.SQL_PASSWORD,
-  // });
-  const teamsfx = new TeamsFx();
-  // If there's only one SQL database
-  const config = await getTediousConnectionConfig(teamsfx);
-  // If there are multiple SQL databases
-  const config2 = await getTediousConnectionConfig(teamsfx, "your database name");  
-  ```
+     ```typescript
+     // Equivalent to:
+     // const sqlConnectConfig = new DefaultTediousConnectionConfiguration({
+     //    sqlServerEndpoint: process.env.SQL_ENDPOINT,
+     //    sqlUsername: process.env.SQL_USER_NAME,
+     //    sqlPassword: process.env.SQL_PASSWORD,
+     // });
+     const teamsfx = new TeamsFx();
+     // If there's only one SQL database
+     const config = await getTediousConnectionConfig(teamsfx);
+     // If there are multiple SQL databases
+     const config2 = await getTediousConnectionConfig(teamsfx, "your database name");  
+      ```
 
   2. Connect to your database.
 
-  ```typescript
-  const connection = new Connection(config);
-  connection.on("connect", (error) => {
-    if (error) {
+     ```typescript
+     const connection = new Connection(config);
+     connection.on("connect", (error) => {
+     if (error) {
       console.log(error);
-    }
-  });  
-  ```
+      }
+     });  
+      ```
 
   > [!NOTE]
-  > For more information on sample to access SQL database in Azure function, see [share-now sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/share-now).
+  > For more information on sample to access SQL database in Azure Function, see [share-now sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/share-now). The
+  > `getTediousConnectionConfig` function has been deprecated, we recommend you to compose your own tedious configuration for better flexibility.
```

Comment:
The
  > `getTediousConnectionConfig` function ...

->
  > The `getTediousConnectionConfig` function